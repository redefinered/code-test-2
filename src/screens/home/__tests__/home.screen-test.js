import React from 'react';
import HomeScreen from '../home.screen';

import renderer from 'react-test-renderer';

describe('Home screen tests', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
