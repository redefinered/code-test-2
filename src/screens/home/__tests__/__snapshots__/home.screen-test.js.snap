// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home screen tests renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#F3F3F3",
    }
  }
>
  <View
    style={
      Object {
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      }
    }
  >
    <TextInput
      autoFocus={true}
      clearButtonMode="always"
      onChangeText={[Function]}
      returnKeyType="search"
      style={
        Object {
          "borderRadius": 10,
          "borderWidth": 1,
          "marginBottom": 10,
          "padding": 10,
        }
      }
      value={Array []}
    />
    <RCTScrollView
      data={Array []}
      getItem={[Function]}
      getItemCount={[Function]}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
    >
      <View />
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
